packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -13,6 +13,14 @@ import type AppRegistry from 'hadron-app-registry';
 
 const toolbarStyles = css({
   padding: spacing[3],
```

Comment:
Looks like this spacing is a bit less in the mocks:
https://www.figma.com/file/vbjoD4dcisPYRYJ5RRu72k/Compass-LG?node-id=2430%3A66209

How about:
```suggestion
  padding: spacing[2],
```